app/views/forms/_working_form.html.erb

```diff
@@ -38,51 +39,68 @@
         </div>
       </div>
       <div class="col-6">
-        <div class="row">
+        <div class="d-flex">
           <div class="col-4">
             <span>Job Title :</span>
           </div>
           <div class="col-8">
-            <span class="form-header"><%= current_user.role %></span>
+            <span class="form-header"><%= current_user.role.titleize %></span>
           </div>
         </div>
       </div>
     </div>
 
-    <div class="row">
+    <div class="d-flex">
       <div class="col-3">
         <span>Reporting manager :</span>
       </div>
       <div class="col-9">
         <span><%= current_user&.manager&.name %></span>
-        <%= form.hidden_field :form_id, value: @form.id %>
+        <%= form.hidden_field :_type, value: @form._type %>
+        <%= form.hidden_field :status, value: 'pending' %>
       </div>
     </div>
 
+    <div class="d-flex">
+      <%= form.file_field :images, multiple: true %>
+    </div>
+
     <table class="table table-bordered mt-4 form-table">
       <thead>
       <tr>
         <% @form.data.keys.each do |key| %>
           <% unless ["reporting_manager"].include? key %>
-            <th scope="col"> <%= key %> </th>
+            <% if @form.id == 2 %>
+              <th scope="col" class="bg-secondary"> <%= key.titleize %> </th>
+            <% else %>
+              <th scope="col" class="bg-secondary"> <%= key.titleize %> </th>
+            <% end %>
           <% end %>
         <% end %>
       </tr>
       </thead>
-      <tbody>
+      <tbody id="working-table">
       <tr>
         <% @form.data.each do |key, value| %>
           <% unless ["reporting_manager"].include? key %>
-            <th scope="col">
-              <%= form.text_field "data[#{key}]", class: "border-none" %>
+            <td>
+              <% if key == "amount" %>
+                <%= form.text_field "data[#{counter}][#{key}]", class: "border-none", type: "number", required: true %></td>
+              <% else %>
+                <%= form.text_field "data[#{counter}][#{key}]", class: "border-none", required: true %></td>
+              <% end %>
```

Comment:
this code is repeating again and again why we have not make partial of all forms?

Reply:
ternary
type: "#{key == "amount" ? 'number' : 'text'}"